### PR TITLE
setup cypress configuration

### PR DESCRIPTION
**Pull request by AlexanderIbarra**
:+1:

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (mateoescobarm) [inline cypress/fixtures/example.json]: for purposes of reading facility and clean code we usually add an aditional line at the end of every file, please add it here

**Review by mateoescobarm — APPROVED** [review]
LGTM

File: cypress/fixtures/example.json (hunk 1 above)
Comment (AlexanderIbarra) [reply cypress/fixtures/example.json]: Sorry this file is autogenerated and i didn't realize it

Comment (AlexanderIbarra) [reply cypress/fixtures/example.json]: I decided to delete this file, it is not important because it is an example

**Pull request merged**